src/d3a/models/market/__init__.py

```diff
@@ -187,18 +187,11 @@ def __repr__(self):  # pragma: no cover
         )
 
     @staticmethod
-    def sorting(obj, reverse_order=False):
-        if reverse_order:
-            # Sorted bids in descending order
-            return list(reversed(sorted(
-                obj.values(),
-                key=lambda b: b.energy_rate)))
-
-        else:
-            # Sorted bids in ascending order
-            return list(sorted(
-                obj.values(),
-                key=lambda b: b.energy_rate))
+    def sorting(offers_bids: Dict, reverse_order=False) -> List[Union[Bid, Offer]]:
+        """Sort a list of bids or offers by a energy_rate attribute."""
+        return list(sorted(offers_bids.values(),
```

Comment:
Why is this forced to list?

Reply:
I thought the sorted function returns a sorted iterable object (like the filter function)
I checked it again and it seems the returned type is a list, I will remove the redundant parsing